docs/source/installation.rst

```diff
@@ -39,7 +57,7 @@ To update your ``Eureka!`` installation to the most recent version, you can do t
 	pip install --no-deps --upgrade .
 
 
-b) With ``pip``
+c) With ``pip``
```

Comment:
Can you please add the `-e` option to this installation method as well

Reply:
Aside from that, as long as the automated pytest run below works successfully, I think this PR is ready to go